bridge/launchpad/import.go

```diff
@@ -70,6 +73,26 @@ func (li *launchpadImporter) ImportAll(repo *cache.RepoCache) error {
 			fmt.Println("TODO: Update bug")
 		}
 
+		/* Handle messages */
+		// The Launchpad API returns the bug description as the first
+		// comment, so skip it.
+		for _, lpMessage := range lpBug.Messages[1:] {
```

Comment:
I would add a check just before to make sure there is actually at least one element here. It would still be an import failure, but at least a polite failure ;)

Reply:
I'm so focused on learning Go that I forgot about the basics :) Nice catch.